charts/kubebb-core/templates/pipeline.yaml

```diff
@@ -0,0 +1,136 @@
+{{- if and (.Values.deployment.rating_enable) (.Capabilities.APIVersions.Has "tekton.dev/v1beta1" ) }}
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: {{ .Release.Name }}-rback
+  namespace: {{ .Release.Namespace }}
+spec:
+  params:
+  - name: url
+    type: string
+  - name: component
+    type: string
+  - name: version
+    type: string
+  - name: repository
+  results:
+  - name: RBACCM
+    type: string
+  steps:
+    - name: rback
+      image: alpine/k8s:1.27.4
+      script: |
+        #!/usr/bin/env sh
+        # Download and untar chart package
+        helm pull --untar $(params.url)
+
+        cat <<EOF | jq --argjson json "$(helm template $(params.component)|yq -o=json|jq --slurp)" '.items |= $json' | rback > r.dot
+        {"kind": "List","apiVersion": "v1","metadata": {},"items": []}
+        EOF
+        
+        # Path to ServiceAccount token
+        SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
+        # Read this Pod's namespace
+        NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
+        NAME=$(params.repository)"."$(params.component)"."$(params.version)
+
+        cat <<EOF |jq --arg dot "$(cat r.dot |base64)" '.binaryData.r = $dot' | jq --arg name "$NAME" '.metadata.name = $name' | jq --arg namespace "$NAMESPACE" '.metadata.namespace = $namespace' | kubectl apply -f -
+        {"kind": "ConfigMap","apiVersion": "v1","metadata": {"name": "configmap","namespace": "default"},"binaryData": {"r": ""}}
+        EOF
+        echo ${NAME} | tee $(results.RBACCM.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: {{ .Release.Name }}-helm-lint
+  namespace: {{ .Release.Namespace }}
+spec:
+  params:
+  - name: url
+    type: string
+  - name: component
+    type: string
+  - name: version
+    type: string
+  results:
+  - name: LINT
+    type: string
+  steps:
+    - name: helm-lint
+      image: alpine/k8s:1.27.4
+      script: |
+        #!/usr/bin/env sh
+        # Download and untar chart package
+        helm pull --untar $(params.url)
+
+        helm lint ./$(params.component)
+        echo $? | tee $(results.LINT.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name:  {{ .Release.Name }}
```

Comment:
We have two pipelines now 
- [component-reliability](https://github.com/kubebb/core/blob/main/pipeline/rating/reliability/pipeline.yaml)
- [component-security](https://github.com/kubebb/core/blob/main/pipeline/rating/security/pipeline.yaml)